docs/user-guide/tracing-tutorial.md

```diff
@@ -59,25 +63,22 @@ spec:
         ports:
         - name: http
           containerPort: 9411
-        resources:
-          limits:
-            cpu: "1"
-            memory: 256Mi
-          requests:
```

Comment:
@danielbryantuk are you facing problems with resources and limits set? We had removed this when it was only limits, but then added it back with requests, so it does not fail.